### PR TITLE
chore: remove tox

### PR DESCRIPTION
**Pull request by JacobCoffee**
# PR Checklist

- [x] Have you followed the guidelines in `CONTRIBUTING.md`?
- [x] Have you got 100% test coverage on new code?
- [x] Have you updated the prose documentation?
- [x] Have you updated the reference documentation?

### CODE REVIEW

**Comment by JacobCoffee** [comment]
I am confused by the test failure

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ To contribute code changes or update the documentation, please follow these step
 5. Create a pull request to the main repository with an explanation of your changes. The PR should detail the
    contribution and link to any related issues - if existing.
 
-## Code contribution guidelines
+## Code Contribution Guidelines
```
Comment (cofin) [inline CONTRIBUTING.md:14]: I'm guessing that pre-commit made these updates?

**Comment by cofin** [comment]
I restarted the action and it looks like it will complete this time.

**Review by cofin — APPROVED** [review]
LGTM

File: CONTRIBUTING.md (hunk 1 above)
Comment (JacobCoffee) [reply CONTRIBUTING.md:14]: I re-pascal-cased the heading as per a discussion in discord, but they can be reverted. 😄

**Comment by cofin** [comment]
@all-contributors please add @JacobCoffee @odiseo0   code, tests, docs

**Comment by cofin** [comment]
https://github.com/all-contributors please add  @odiseo0 code, tests, docs

**Pull request merged**